piet-gpu-hal/Cargo.toml

```diff
@@ -7,12 +7,12 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [dependencies]
-ash = "0.33"
-ash-window = "0.7"
-raw-window-handle = "0.3"
-bitflags = "1.2.1"
-smallvec = "1.6.1"
-bytemuck = "1.7.2"
+ash = "0.37"
+ash-window = "0.11"
+raw-window-handle = "0.5"
+bitflags = "1.3.2"
```

Comment:
Do we need to specify the minor version here? See also for other crates in this and piet-gpu's Cargo.toml

Reply:
We don't need to, but I generally do.